addons/website_slides/models/slide_channel.py

```diff
@@ -528,13 +527,18 @@ def message_post(self, *, parent_id=False, subtype_id=False, **kwargs):
     # ---------------------------------------------------------
 
     def action_redirect_to_members(self, completed=False):
-        """ Redirects to attendees of the course. If completed is True, a filter
-        will be added in action that will display only attendees who have completed
-        the course. """
-        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
         action_ctx = {'active_test': False}
+        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
         if completed:
             action_ctx['search_default_filter_completed'] = 1
+        action['domain'] = [('channel_id', 'in', self.ids)]
+        help_message = [
+            """No Attendees Yet!""",
```

Comment:
this is the translatable content where `_()` should be used, not the empty html below